PyInstaller/compat.py

```diff
@@ -39,7 +39,8 @@
 
 is_win = sys.platform.startswith('win')
 is_win_10 = is_win and (platform.win32_ver()[0] == '10')
-is_cygwin = sys.platform == 'cygwin'
+is_cygwin = sys.platform == 'cygwin' 
```

Comment:
What has been changed on this line? I can not spot it.

Reply:
Nothing as far as I remember: I deleted the line and then ended up re-adding it? Maybe a line ending thing?

Reply:
There's an extra space been added at the end of the line.